packages/react-accordion/src/components/AccordionHeader/AccordionHeader.types.ts

```diff
@@ -23,15 +23,15 @@ export type AccordionHeaderSlots = {
   /**
    * The component to be used as button in heading
    */
-  button: NonNullable<Slot<ARIAButtonSlotProps>>;
+  button: Slot<ARIAButtonSlotProps>;
   /**
    * Expand icon slot rendered before (or after) children content in heading
    */
   expandIcon: Slot<'span'>;
```

Comment:
Note: this slot was previously nullable, but with this change it is non-nullable (as a result of `Slot` no longer including `| null`). It appears from the code that it was never intended to be nullable.